server/middleware/setUpAuthentication.ts

```diff
@@ -12,7 +11,6 @@ export default function setUpAuth(): Router {
 
   router.use(passport.initialize())
   router.use(passport.session())
-  router.use(flash())
```

Comment:
Fine for this to go now, but flash is really useful when using post/redirect/get which would generally recommend services to use rather than write their own session management abstraction to handle this use case.

We might add a form example using classvalidator in which case this might be re-added